recipes/exparna/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: exparna
+  version: 1.0.1
+
+source:
+  git_url: https://github.com/BackofenLab/ExpaRNA.git
```

Comment:
How can I merge my updated master with this branch? git merge master/exparna?
Because this branch is a few days old.